src/tabixIndexedFile.ts

```diff
@@ -236,22 +240,25 @@ export default class TabixIndexedFile {
             // then the blockStart-dpositions is an uncompressed file offset from
             // that bgzip block boundary, and since the cpositions are multiplied by
             // (1 << 8) these uncompressed offsets get a unique space
-            cpositions[pos] * (1 << 8) + (blockStart - dpositions[pos]),
+            cpositions[pos] * (1 << 8) +
```

Comment:
this syncs up with equivalent bam code also https://github.com/GMOD/bam-js/blob/master/src/bamFile.ts#L499